### ISSUE TITLE
MCPC L29 or higher and tickthreading not working

### ISSUE THREAD

**Issue by Maximuscr31** [MinimallyCorrect/TickThreading#573]
I don't know why but I can't install any mcpc 29 or higher. I have tried 29+ with tickthreading 1634 to 1670 with no luck. I can install any tickthreading with L28 or below fine. I can install mcpc 29+ fine without tickthreading. Add in tickthreading and it crashes while loading mods. I have checked tickthreading logs, forgecraft logs, crash reports, and server log. There is nothing that explains it. It just says server stopped, restarting. It does it right after trying to load thermal expansion on ultimate. We are using chickenchunks 1.3.1.2 instead of the standard. I have removed all plugins and it don't matter because it doesn't get to them. Same with the world. Any idea what would cause tickthreading to freak out and stop the server like this? I really would love to go to mcpc 40+ to fix xycraft tanks. I also redownloaded and rebuilt the entire server with fresh files and still wouldn't work. Any guidance or info that might help just let me know.

**Comment by LunNova** [comment]
Post FML and tickthreading logs.

**Comment by Maximuscr31** [comment]
https://drive.google.com/folderview?id=0B5yKfXm1PkuLclA0VWJ3c1BxdzA&usp=sharing
They are in the test folder there. 
I deleted them all before the patching, patched, then started it so it would all be fresh and no clutter. That is trying the latest MCPC l40

**Comment by LunNova** [comment]
Works fine for me,  broken for you as you're 45 builds of TT out of date. Fixed a while ago.

Closed as duplicate of #545.

**Comment by Maximuscr31** [comment]
Feel like an idiot now. I updated to 1700+ after that but I had made the assumption it was something else because when i was on 1630-1670 other people were posting tickets using l30+. I had downgrade to 1645 because I thought the newer version was the cause of another problem and it wasn't. I just didn't get it redone till after I sent you those logs. Thanks for the help and everything is great now.
